### DISCUSSION TITLE
No config file specified

### DISCUSSION THREAD

**Discussion by JCionx** [v1993/linuxmotehook2#8] [General]
Hi! Today I just installed linuxmotehook2 on my computer to play on Cemu with a Wiimote. But when I run the program, it tells me this "No config file specified - call with `--help' for usage information." How can I create a config file?

**Comment by v1993** [comment]
You can refer to included `ExampleConfig.ini` for general config structure. I'd suggest to start with an empty file and continue from there.

**Comment by JCionx** [comment]
> You can refer to included `ExampleConfig.ini` for general config structure. I'd suggest to start with an empty file and continue from there.

But where and how to create the file? (Sorry, I'm completely new to linux)

**Reply by v1993** [reply]
I threw together a little guide for config file in the meantime: https://github.com/v1993/linuxmotehook2/wiki

It does not matter where you place config file - just pass its path as an argument to the program. It can be created with any text editor of your choice.

**Reply by JCionx** [reply]
Thank you! I got it working now.

**Comment by JCionx** [comment]
Is there any way of using the sensor bar on Cemu? I tried to play Wii Party U, but the pointer is not shown.

**Reply by v1993** [reply]
Unfortunately, no. This is a limitation of underlying DSU protocol itself.
